### PR TITLE
Some general cleanup for haraka

### PR DESCRIPTION
**Pull request by zero-24**
Hi @brianteeman 

here are some some general cleanup for:
- update copyright
- remove some not needed variables
- sync the docblock style over all the language files

### CODE REVIEW

File: haraka.php [hunk 1]
```diff
@@ -68,45 +67,39 @@ private function displayHaraka()
...
-			$fonts          = $this->params->get('fonts', 'Roboto+Slab|Roboto');
-			$font           = explode("|", $fonts);
-			$fontcss        = str_replace('+',' ', $font);
-			$text           = $this->params->get('text', '<p>' . JText::_('PLG_SYSTEM_HARAKA_COMING_SOON') . '</p>');
-			$theme          = $this->params->get('theme', 'light');
-			$uri            = JUri::getInstance();
+			$bgimageUrl    = JUri::base() . 'media/plg_haraka/images/' . $this->params->get('bgimage', 'comingsoon.jpg');
+			$caption       = $this->params->get('caption', '');
+			$countdown     = $this->params->get('countdown', 1);
+			$countdownDate = $this->params->get('countdown_date', '');
```
Comment (brianteeman) [inline haraka.php]: I deliberately use the _ as to me it is more readable

File: haraka.php [hunk 2]
```diff
@@ -68,45 +67,39 @@ private function displayHaraka()
...
-			$instagram     = $this->params->get('instagram', '');
-			$instagram_url = 'https://instagram.com/' . $instagram;
-			$twitter       = $this->params->get('twitter', '');
-			$twitter_url   = 'https://twitter.com/' . $twitter;
-			$youtube       = $this->params->get('youtube', '');
-			$youtube_url   = 'https://youtube.com/' . $youtube;
+			$facebook  = $this->params->get('facebook', '');
+			$instagram = $this->params->get('instagram', '');
+			$twitter   = $this->params->get('twitter', '');
+			$youtube   = $this->params->get('youtube', '');
```
Comment (brianteeman) [inline haraka.php]: I deliberately has the url as a variable in case it changes in the future

**Comment by zero-24** [comment]
done @brianteeman

File: haraka.php [hunk 3]
```diff
@@ -68,46 +67,46 @@ private function displayHaraka()
...
+
+			// Social Media URL's
 			$facebook_url  = 'https://facebook.com/' . $facebook;
-			$instagram     = $this->params->get('instagram', '');
-			$instagram_url = 'https://instagram.com/' . $instagram;
-			$twitter       = $this->params->get('twitter', '');
-			$twitter_url   = 'https://twitter.com/' . $twitter;
-			$youtube       = $this->params->get('youtube', '');
+			$instagram_url = 'https://twitter.com/' . $instagram;
+			$twitte_url    = 'https://instagram.com/' . $twitter;
```
Comment (brianteeman) [inline haraka.php]: twitter - you missed an r

File: tmpl/default.php [hunk 4]
```diff
@@ -16,14 +16,14 @@
...
...{font-size:1.875em;line-height:1.33em}#counter{margin-top:3.75em;float:left}.countdown_section{color:#fff;display:inline-block;text-align:center;width:15%;box-sizing:border-box;font-size:.938em;text-transform:lowercase;font-style:italic;padding:2.667em 0;border-top:2px solid rgba(255,255,255,.2);border-bottom:2px solid rgba(255,255,255,.2)}.countdown_section:first-child{border-left:0}.countdown_amount{color:#fff;display:block;font-size:3.33em;font-weight:700;font-style:normal;line-height:.75em}.social{width:40%;float:left;padding:0 30%;margin:3.125em 0 0}.social ul li{display:inline;margin:0 1.25em}.social ul li a{color:#fff}@media only screen and (max-width:1024px){body{font-size:90%}.content{width:80%;padding:0 10%}#counter,.social{width:60%;float:left;padding:0 20%}#counter{margin-top:3.75em}.social{margin:3.125em 0 0}}@media only screen and (max-width:768px){.content p,h1{text-shadow:1px 1px 3px #000}#counter,.content,.social{width:90%;padding:0 5%}body{font-size:80%}h1{margin:.3em 0 .6em;font-size:5.25em;line-height:.9em}.content p{font-size:1.475em;line-height:1.63em}#counter{margin-top:3.75em;float:left}.social{float:left;margin:2.125em 0 0}.social ul{width:100%;float:left}}
```
Comment (brianteeman) [inline tmpl/default.php:24]: what is the change here?

Comment (mikeveeckmans) [reply tmpl/default.php:24]: $bgimageUrl

Comment (brianteeman) [reply tmpl/default.php:24]: good spot. I just couldn't see it

**Comment by brianteeman** [comment]
Thanks @zero-24 

I wont do a new release just yet

**Comment by zero-24** [comment]
No problem :)

**Comment by zero-24** [comment]
Thanks for merging

**Pull request merged**